ShopifySharp/Entities/FulfillmentShipping.cs

```diff
@@ -0,0 +1,37 @@
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// An object representing a Shopify fulfillment request.
```

Comment:
Looks like the comment here is wrong. Should say it represents fulfillment shipping options?